pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "2.4.0b1"
+version = "3.0.0a1"
```

Comment:
Should also create a `version-3.0.md` release note and add it to `mkdocs.yml` at some point.